2020/join/gatsby-new-join/src/components/business.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component, useState } from "react"
```

Comment:
use single quotation mark instead of double quotation mark. I.e. `import React, { Component, useState } from 'react'`